gst/nnstreamer/tensor_common.h

```diff
@@ -108,6 +108,33 @@ typedef struct
   guint nth;
 } GstTensorPad;
 
+/**
+ * @brief Sets the value of a property based on the specified property value and GParamSpec.
+ *
+ * @param prop_value A pointer to the GValue where the property value will be set.
+ * @param param_spec A pointer to the GParamSpec that describes the property.
+ * @param property_value A string representing the value to be set for the property.
+ *
+ * @note This API is intended to be used by gst_tensor_parse_config_file ()
+ */
+void
+set_property_value (GValue * prop_value, const GParamSpec * param_spec, const gchar * property_value);
```

Comment:
If this function is used only in gst_tensor_parse_config_file (). Let's move this to nnstreamer_plugin_api_impl.c and makes as a static function.

Reply:
I will update this. Thank you